packages/main/src/components/ObjectPage/index.tsx

```diff
@@ -705,9 +705,9 @@ const ObjectPage: FC<ObjectPagePropTypes> = forwardRef((props: ObjectPagePropTyp
         </div>
       )}
       <div
+        className={classes.anchorBar}
         data-component-name="ObjectPageTabContainer"
         style={{
-          position: 'sticky',
```

Comment:
We need `zIndex: 1` here, otherwise the lower part of the buttons is cut off. In my opinion it would make sense to create a new style class for this. WDYT
<img width="1079" alt="Screenshot 2021-07-22 at 22 56 37" src="https://user-images.githubusercontent.com/9749730/126711923-3bf0f73b-0ec5-4474-9c82-3289ec4bbc35.png">

Reply:
you're absolutely right :) done